### DISCUSSION TITLE
tox.ini > [testenv] section > deps: Where is the code that handles deps?

### DISCUSSION THREAD

**Discussion by samuelwhiskeyjohnson** [tox-dev/tox#3298] [Q&A] [answered]
I realized you can do 
`deps = -rrequirements.txt`
or `deps = -r requirements.txt` (which was not possible before in tox 3)

Where is the location of the code that handles parsing tox.ini and automatically adds space if you wrote `-rrequirements.txt`?

**Comment by gaborbernat** [comment, marked as answer]
https://github.com/tox-dev/tox/blob/main/src%2Ftox%2Ftox_env%2Fpython%2Fpip%2Freq_file.py#L53

**Reply by samuelwhiskeyjohnson** [reply]
Thanks so much
